msnoise/s03compute_cc.py

```diff
@@ -116,6 +116,10 @@ def preprocess(db, stations, comps, goal_day, params, tramef_Z, tramef_E = None,
 
                 logging.debug("Checking sample alignment")
                 for i, trace in enumerate(stream):
+                    ## when traces are read as ints and some are modified in check_and_phase_shift
+                    ## data types differ and traces can't be concatenated
+                    trace.data = trace.data.astype(np.float64, copy=False)
```

Comment:
I noticed after adding this that it appears the typecasting was done above already! So please disregard; I'll let you know if I have questions.